source/D2CommonDefinitions/include/D2PacketDef.h

```diff
@@ -383,8 +388,8 @@ struct D2GSPacketClt44		//size of 0x11
 struct D2GSPacketClt45		//size of 0x09
 {
 	uint8_t nHeader;			//0x00
-	int32_t unk0x01;			//0x01
-	int32_t unk0x05;			//0x05
+	int32_t nPortalGUID;			//0x01
```

Comment:
Comments are no longer aligned (tabspace = 4), this can not be seen by default as github uses 8 spaces for tabs.
I really need to get some .clang-format setup, but it quickly messes up with array constants so I need to investigate more before doing so...

Note for myself in the future: this can be changed at the user level in https://github.com/settings/appearance

Reply:
I vaguely remember github respecting .editor or .editorconfig in the root of the repo for display purposes.

Reply:
Indeed:
https://github.com/isaacs/github/issues/170#issuecomment-150489692

Reply:
It's kind of unfortunate we have to add yet another file for this :(
But I guess this is better than having to explain the issue every time.